guides/common/modules/proc_using-container-registries.adoc

```diff
@@ -2,31 +2,73 @@
 = Using container registries
 
 Podman and Docker can be used to fetch content from container registries.
+Containers are pushed to {Project}'s container registry.
+The {Project} registry follows the Open Containers Initiative (OCI) specification, so you can push content to Katello by using the same methods that apply to other registries.
+For more information about OCI, see https://opencontainers.org/[Open Container Initiative Distribution Specification].
+
+.Prerequisites
+* To push content to an organization or product, a user must have permissions to create repositories.
+The push permission is `edit_products`.
+* To pull content, a user must have permissions as long as *unauthenticated pull* is set to off by default in the lifecycle environment.
+The pull permissions are `view_lifecycle_environments`, `view_products`, `view_content_views`.
 
 ifndef::orcharhino[]
 .Container registries on {SmartProxies}
-
 On {SmartProxies} with content, the https://github.com/Katello/smart_proxy_container_gateway[Container Gateway] {SmartProxy} plugin acts as the container registry.
 It caches authentication information from Katello and proxies incoming requests to Pulp.
 The Container Gateway is available by default on {SmartProxies} with content.
 endif::[]
 
-.Procedure
+.Limitations of pushing content to the {Project} container registry
```

Comment:
```suggestion
.Considerations for pushing content to the {Project} container registry
```
or

```suggestion
.Guidelines for pushing content to the {Project} container registry
```

I'm thinking of how to frame this in a more neutral light, to avoid making it sound like the feature is not working properly. What do you think @bangelic, would this describe the purpose of the list accurately enough?

Reply:
@asteflova I think I like **Considerations** better :)